lib/go/test/go.mod

```diff
@@ -3,12 +3,12 @@ module github.com/onflow/lib/go/flow-core-contracts
 go 1.14
 
 require (
-	github.com/onflow/cadence v0.12.3
-	github.com/onflow/flow-core-contracts/lib/go/contracts v0.6.1-0.20210118202839-f8f9d9a04bcb
+	github.com/onflow/cadence v0.12.5
+	github.com/onflow/flow-core-contracts/lib/go/contracts v0.7.1
 	github.com/onflow/flow-core-contracts/lib/go/templates v0.0.0-00010101000000-000000000000
-	github.com/onflow/flow-emulator v0.14.0-beta3
+	github.com/onflow/flow-emulator v0.14.0
```

Comment:
Maybe update to [v0.14.2](https://github.com/onflow/flow-emulator/releases/tag/v0.14.2)